examples/kubernetes/quickstart.yaml

```diff
@@ -15,6 +15,12 @@ spec:
     - name: http
       protocol: HTTP
       port: 8080
+    - name: tls
+      protocol: TLS
+      hostname: passthrough.example.com
+      tls:
+        mode: Passthrough
+      port: 8443
```

Comment:
Due to https://github.com/envoyproxy/gateway/issues/520 and https://github.com/envoyproxy/gateway/pull/542 please use a different port number.

Reply:
using 6443 now